### PR TITLE
Updating Protostomes and Default GOC configs

### PR DESCRIPTION
**Pull request by Simarpreet-Kaur-Bhurji**
Created Protostomes GOC configs and updated default GOC configs. GOC heat plots and grid plots were created and compared to identify the GOC taxlevels, thresholds for Protostomes and Default collections. GOC configuration has been updated for protostomes and default Metazoa collections with values appropriate to the species in those collections.

ENSCOMPARASW-6368 

---

For code reviewers: [code review SOP](https://www.ebi.ac.uk/seqdb/confluence/display/EnsCom/Code+review+SOP)

### CODE REVIEW

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Metazoa/ProteinTrees_conf.pm [hunk 1]
```diff
@@ -48,36 +48,28 @@ sub default_options {
...
         # In this structure, the "thresholds" are for resp. the GOC score, the WGA coverage and %identity
+
         'threshold_levels' => [
             {
-                'taxa'          => [ 'Euteleostomi', 'Ciona' ],
-                'thresholds'    => [ 50, 50, 25 ],
-            },
-            {
-                'taxa'          => [ 'Aculeata', 'Anophelinae', 'Caenorhabditis', 'Drosophila', 'Glossinidae', 'Onchocercidae' ],
+                'taxa'          => [ 'Penaeus', 'Thoracicalcarea', 'Dhapnia', 'Apinae', 'Haliotis', 'Echinacea', 'Asteroidea', 'Anthozoa', 'Scleractinia' ],
```
Comment (twalsh-ebi) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Metazoa/ProteinTrees_conf.pm:61]: Could the name of the third taxon from the left here include a typo? Should it be spelled `Daphnia`?

Comment (twalsh-ebi) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Metazoa/ProteinTrees_conf.pm:61]: Could the name of the third taxon from the left here include a typo? Should it be spelled `Daphnia`?
```suggestion
                'taxa'          => [ 'Penaeus', 'Thoracicalcarea', 'Daphnia', 'Apinae', 'Haliotis', 'Echinacea', 'Asteroidea', 'Anthozoa', 'Scleractinia' ],
```

Comment (ens-LCampbell) [reply modules/Bio/EnsEMBL/Compara/PipeConfig/Metazoa/ProteinTrees_conf.pm]: It does appear to be misspelled

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Metazoa/ProteinTrees_conf.pm [hunk 2]
```diff
@@ -48,36 +48,28 @@ sub default_options {
...
         # In this structure, the "thresholds" are for resp. the GOC score, the WGA coverage and %identity
+
         'threshold_levels' => [
             {
-                'taxa'          => [ 'Euteleostomi', 'Ciona' ],
-                'thresholds'    => [ 50, 50, 25 ],
-            },
-            {
-                'taxa'          => [ 'Aculeata', 'Anophelinae', 'Caenorhabditis', 'Drosophila', 'Glossinidae', 'Onchocercidae' ],
+                'taxa'          => [ 'Penaeus', 'Thoracicalcarea', 'Daphnia', 'Apinae', 'Haliotis', 'Echinacea', 'Asteroidea', 'Anthozoa', 'Scleractinia' ],
```
Comment (ens-LCampbell) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Metazoa/ProteinTrees_conf.pm:61]: https://github.com/Ensembl/ensembl-compara/blob/42c698bcd17f95efa1e4d59aff99a21a1adf10e3/conf/metazoa/mlss_conf.xml#L96C13-L96C13

Would it be a good idea to add Parachela to this group  to account for the two tardigrade species ? 
https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=149990&lvl=3&lin=f&keep=1&srchmode=1&unlock

**Review by twalsh-ebi — APPROVED** [review]

**Review by ens-LCampbell — APPROVED** [review]
Thanks for the updates to parachela Simarpreet looks good to me !

**Pull request merged**